src/vxworks/mod.rs

```diff
@@ -2013,6 +2029,35 @@ pub unsafe fn posix_memalign(
     }
 }
 
+pub unsafe fn flock(fd: ::c_int, operation: ::c_int) -> ::c_int {
```

Comment:
We prefer not to have any polyfill implementations in `libc`, only interfaces as much as possible. It would be best to drop this function from `libc` and instead add this implementation wherever this would be called - for `std` this is at https://github.com/rust-lang/rust/blob/76fd47124b8f8037b6187169b2cdf39139466952/library/std/src/sys/pal/unix/fs.rs#L1268-L1276.